fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/digitalcash/DigitalCashSendFragment.kt

```diff
@@ -148,14 +151,20 @@ class DigitalCashSendFragment : Fragment() {
     binding.digitalCashSendSpinnerTv.setAdapter(adapter)
   }
 
+  private fun getPublicKeyFromUsername(username: String): PublicKey {
+    return digitalCashViewModel.attendeesFromTheRollCallList.firstOrNull {
+      it.getUsername() == username
+    } ?: digitalCashViewModel.validToken.publicKey
+  }
+
```

Comment:
I see this function repeated across different files -> refactoring opportunity: create a static method that takes username, list of public key and alternative option if null. As the place I'd say either the PublicKey class itself or an utils class